aws/cloudformation/domain_redirect.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
I'd love to see a usage example for this script somewhere, with example target and domain commands. Either here in a comment in the top of the file, in the "invalid option" error message, or in a `-h/--help` option

Reply:
I added -h, with a note to check it when input errors occur.